src/python_testing/matter_testing_support.py

```diff
@@ -238,7 +238,7 @@ def __init__(self, expected_cluster: ClusterObjects.Cluster):
         self._q = queue.Queue()
         self._expected_cluster = expected_cluster
 
-    async def start(self, dev_ctrl, node_id: int, endpoint: int, fabric_filtered: bool = False, min_interval_sec: int = 0, max_interval_sec: int = 30) -> Any:
+    async def start(self, dev_ctrl, node_id: int, endpoint: int, fabric_filtered: bool = False, min_interval_sec: int = 0, max_interval_sec: int = 5) -> Any:
```

Comment:
Why 5 instead of 30? Battery powered  Thread switches may not be able to do 5. Suggest keeping 30

Reply:
If the ceiling is 30 and we're waiting for 30 seconds, we might not get reports at all. Device can adjust the ceiling - this is just a request.